config.go

```diff
@@ -13,6 +13,9 @@ type Config struct {
 
 	AWSAccessKeyID     string `help:"the access key id to use when authenticating S3"`
 	AWSSecretAccessKey string `help:"the secret access key id to use when authenticating S3"`
+
+	ArchiveTempDirectory string `help:"directory where temporary archive files are written"`
```

Comment:
Just `TempDir` is better for no other reason than the env variable will turn into `ARCHIVER_TEMP_DIR` instead of `ARCHIVER_ARCHIVE_TEMP_DIRECTORY` which is a mouthful.